profile/E-ARK-SIP.xml

```diff
@@ -215,14 +216,14 @@
                     </dl>
                 </description>
             </requirement>
-            <requirement ID="SIP12" REQLEVEL="MAY" EXAMPLES="metsHdrElementExample1 metsHdrAgentExample1">
+            <requirement ID="SIP12" REQLEVEL="MUST" EXAMPLES="metsHdrElementExample1 metsHdrAgentExample1">
                 <description>
                     <head>Archival creator agent name</head>
                     <p xmlns="http://www.w3.org/1999/xhtml">The name of the organisation(s) that originally created the data being transferred.</p>
                     <p xmlns="http://www.w3.org/1999/xhtml">Please note that this might be different from the organisation which has been charged with preparing and sending the SIP to the archives.</p>
                     <dl xmlns="http://www.w3.org/1999/xhtml">
                         <dt>METS XPath</dt><dd>metsHdr/agent/name</dd>
-                        <dt>Cardinality</dt><dd>0..*</dd>
+                        <dt>Cardinality</dt><dd>0..1</dd>
```

Comment:
Hope you don't mind me jumping in, this should be `1..1`

Reply:
@karinbredenberg Karin, @prettybits is right. It should be `1..1`.

```xml
<sequence>
	<element name="name" type="xsd:string">
		<annotation>
			<documentation xml:lang="en">
				The element &lt;name> can be used to record the full name of the document agent.
			</documentation>
		</annotation>
	</element>
</sequence>
```

Reply:
Sorry I missed that. Fixed and PR updated.